src/metabase/api/search.clj

```diff
@@ -442,6 +443,47 @@
                    :last_editor_common_name (get user-id->common-name last_editor_id)))
           results)))
 
+(defn add-dataset-collection-hierarchy
+  "Adds collection names to datasets."
```

Comment:
Is this docstring accurate? It's adding `:collection_effective_ancestors` which is more than just the collection names, right?

Reply:
Good call, thanks!